src/plugins/formvalid/formvalid-en.hbs

```diff
@@ -424,7 +424,7 @@
 		&lt;/fieldset&gt;</code></pre>
 			</details>
 
-			<input type="submit" value="Submit" class="btn btn-primary" /> <input type="reset" value="Reset" class="btn btn-default" />
+			<input type="submit" value="Submit" class="btn btn-primary" /> <input type="reset" value="Reset page to defaults" class="btn btn-link pull-right" />
```

Comment:
Not sure regarding to pull the reset button to the right. That would fail the straw test.

Reply:
Made have it under the submit button, but with a decent amount of spacing between them to prevent accidental clicking?